tools/cli_install.py

```diff
@@ -85,6 +85,10 @@ def _get_tempfilename(a_dir):
 def _mark_executable(path):
     os.chmod(path, 0o755)
 
+def install_cli_from_dir(src_dir, write_dir):
+    src_file = os.path.join(src_dir, get_cli_filename())
+    install_cli(src_file, write_dir)
+
```

Comment:
I don't really think this style suggestion matters at all.  However, I'm happy to update the file to do that overall if you care.